lib/tasks/git-init.js

```diff
@@ -53,8 +48,19 @@ class GitInitTask extends Task {
     return execa('git', ['add', '.']);
   }
 
-  _gitCommit(commitMessage) {
-    return execa('git', ['commit', '-m', commitMessage], { env: this.buildGitEnvironment() });
+  _gitCommit() {
+    const template = require('lodash.template');
+    let commitTemplate = fs.readFileSync(path.join(__dirname, '../utilities/COMMIT_MESSAGE.txt'));
+    let commitMessage = template(commitTemplate)(pkg);
+    let env = this.buildGitEnvironment();
+    return execa('git', ['commit', '-m', commitMessage], { env })
+      .catch(error => {
+        if (error && error.message && error.message.indexOf('git config --global user') > -1) {
+          env.GIT_COMMITTER_NAME = 'Tomster';
+          env.GIT_COMMITTER_EMAIL = 'tomster@emberjs.com';
+          return execa('git', ['commit', '-m', commitMessage], { env });
+        }
```

Comment:
I'm so frustrated I didn't handle this properly in the first place... Thanks for fixing the PR!